Framework/Algorithms/inc/MantidAlgorithms/CalculateFlatBackground.h

```diff
@@ -86,24 +83,17 @@ class DLLExport CalculateFlatBackground : public API::Algorithm {
   void convertToDistribution(API::MatrixWorkspace_sptr workspace);
   void restoreDistributionState(API::MatrixWorkspace_sptr workspace);
   void checkRange(double &startX, double &endX);
-  void getWsInds(std::vector<int> &output, const int workspaceTotal);
-  double Mean(const API::MatrixWorkspace_sptr WS, const int wsInd,
-              const double startX, const double endX) const;
-  double LinearFit(API::MatrixWorkspace_sptr WS, int spectrum, double startX,
-                   double endX);
-  double MovingAverage(API::MatrixWorkspace_const_sptr WS, int wsIndex,
-                       size_t windowWidth) const;
-
-  /// variable bin width raw count data must be converted to distributions first
-  /// and then converted back, keep track of this
-  bool m_convertedFromRawCounts;
-  /// the variable which specifies if background should be removed from monitors
-  /// too.
-  bool m_skipMonitors;
-  // if true, negative signals appearing after background removals are set to 0
-  bool m_nullifyNegative;
+  void getWsInds(std::vector<int> &output, const size_t workspaceTotal);
```

Comment:
No. But since this method has become so short with `std::iota`, it might make `exec()` a bit clearer if the code was moved there.